tools/vscode/README.md

```diff
@@ -20,6 +20,19 @@ C/C++ code completion.
 
 For developers in China, you might need to set `http_proxy`, `https_proxy`, `all_proxy` in advance.
 
+If you see the following error message: 
+```
+ERROR: vscode@docker-desktop:/workspaces/<your-username>$ tools/vscode/refresh_compdb.sh
+/usr/bin/env: ‘bash\r’: No such file or directory
```

Comment:
```suggestion
```console
ERROR: vscode@docker-desktop:/workspaces/<your-username>$ tools/vscode/refresh_compdb.sh
/usr/bin/env: ‘bash\r’: No such file or directory
```

Reply:
this code block needs to be closed with ` ``` `